delfin/exporter/prometheus.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metrics_name = ['response_time', 'throughput', 'read_throughput',
+                'write_throughput', 'bandwidth', 'read_bandwidth',
+                'write_bandwidth']
+
+
+class PrometheusExporter(object):
```

Comment:
Did you check promethus_client from Promethus? Can it be used for some of the operations here?

Reply:
existing prometheus client can not be used here. becasue our data model is different.